hack/release.sh

```diff
@@ -18,4 +18,8 @@ do
 	OUTPUT="bin/openshift-install-${GOOS}-${GOARCH}"
 	GOOS="${GOOS}" GOARCH="${GOARCH}" OUTPUT="${OUTPUT}" SKIP_GENERATION=y ./build.sh
 done
-(cd ../bin && sha256sum openshift-install-*)
+(
+	cd ../bin
```

Comment:
why do we need to change the directory?

Reply:
> why do we need to change the directory?

So the paths that `sha256sum` spits out don't include a `bin/` prefix.